desktop-src/w8cookbook/windows-version-check.md

```diff
@@ -23,75 +23,49 @@ Some apps perform a version check and simply pass a warning to users. However, t
 -   If the app is dependent on specific API functionality, ensure you target the correct API version.
```

Comment:
This documentation page is substantially duplicated in 2 places:
- https://docs.microsoft.com/en-us/windows/win32/w8cookbook/windows-version-check (this file)
- https://docs.microsoft.com/en-us/windows/compatibility/windows-version-check

Unfortunately, the other copy of the source file isn't open-source on GitHub as far as I can tell, though I do have internal access to it. I'll submit an internal PR to update it, but the 2 files should really be merged or one deleted altogether.